types/ApiRequests.ts

```diff
@@ -183,6 +183,10 @@ export interface EditMerchCollectionRequest {
   collection: MerchCollectionEdit;
 }
 
+export interface CreateCollectionPhotoRequest {
+  position: string;
```

Comment:
Seeking to learn: why string instead of number?

Reply:
^ i know this is very old but for multi form data request, we can only have types "string" and "file". That means we can only get position input as a string rather than a number and we process that to a number afterward

Reply:
What do you mean by multi form data request? Is this an Express rule?

Reply:
I believe this is a HTTP protocol rule https://github.com/spring-projects/spring-boot/issues/8441#issuecomment-361947755, if you look at the postman request, if you select multipart/form-data, you can only set fields to "file" or "string". In other words, you cannot set a field to number explicitly with multipart request, which is required here to upload an image.